examples/skybox/main.rs

```diff
@@ -364,14 +364,11 @@ impl framework::Example for Skybox {
     }
 
     fn update(&mut self, event: winit::event::WindowEvent) {
-        match event {
-            winit::event::WindowEvent::CursorMoved { position, .. } => {
-                let norm_x = position.x as f32 / self.camera.screen_size.0 as f32 - 0.5;
-                let norm_y = position.y as f32 / self.camera.screen_size.1 as f32 - 0.5;
-                self.camera.angle_y = norm_x * 5.0;
-                self.camera.angle_xz = norm_y;
-            }
-            _ => {}
+        if let winit::event::WindowEvent::CursorMoved { position, .. } = event {
```

Comment:
hmm, that's not desired. if/let is not scaleable: once we need to add a new variant, we'll have to go back to a match